contributors/devel/help-wanted.md

```diff
@@ -1,73 +1,87 @@
+We use two labels `help wanted` and `good first issue` to identify issues that have been
+specially groomed for new and casual contributors. The `good first issue` label is a 
```

Comment:
The more I think of it I'm not sure I like the word "casual" contributor...it might imply carelessness on the contributor's side.

Maybe just say something like:
 "new contributors.  The `good first issue` label is a subset of `help-wanted` for issues that are especially good for folks new to Kubernetes."